src/trackinfoobject.cpp

```diff
@@ -37,12 +9,34 @@
 #include "track/keyutils.h"
 #include "util/compatibility.h"
 #include "util/cmdlineargs.h"
-#include "util/time.h"
 #include "util/math.h"
 #include "waveform/waveform.h"
 #include "library/coverartutils.h"
 #include "util/assert.h"
 
+#include <QDomNode>
+#include <QDomDocument>
+#include <QDomElement>
+#include <QFileInfo>
+#include <QDirIterator>
+#include <QFile>
+#include <QMutexLocker>
+#include <QString>
+#include <QtDebug>
+#include <QRegExp>
```

Comment:
Why did you change the order of the headers? 
We have this rule http://mixxx.org/wiki/doku.php/coding_guidelines#includes
But it is OK to break it for a good reason.